sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/models/HealthcareEntityCategory.java

```diff
@@ -10,83 +10,83 @@
 
 /** Defines values for HealthcareEntityCategory. */
 public final class HealthcareEntityCategory extends ExpandableStringEnum<HealthcareEntityCategory> {
-    /** Static value BODY_STRUCTURE for HealthcareEntityCategory. */
```

Comment:
The explored model is already in the correct value.
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/HealthcareEntityCategory.java